Assets/00_Content/Scripts/Vikings/States/BrawlingVikingState.cs

```diff
@@ -64,6 +100,23 @@ public class BrawlingVikingState : VikingState {
 			return this;
 		}
 
+		private VikingState DoVikingBrawl() {
+			Debug.Log("Fighting another viking...");
+			return this;
+		}
+
+		private VikingState DoPlayerBrawl() {
+
+			if ((navMeshAgent.transform.position - playerTarget.transform.position).sqrMagnitude < viking.Data.attackTriggerDistance && !viking.IsAttacking) {
```

Comment:
You're comparing square distance with normal distance. You should probably do something like `sqrMag < triggerDist * triggerDist` instead, or rename the variable.

Reply:
Yeah agreed, im square!